### ISSUE TITLE
PixateFreestyle doesn't work well in iOS 9 beta  

### ISSUE THREAD

**Issue by markddaa** [Pixate/pixate-freestyle-ios#178]
what I did:
0. I use Xcode 7 beta, simulator iOS 9 beta, and Object-C
1. create a demo project with one button and a UIAlertViewController with textfield
2. add "pod 'PixateFreestyle'" in the podfile
3. run pod install
4. run the app
then I got this
![qq20150702-1](https://cloud.githubusercontent.com/assets/953116/8472277/6ba9a994-20d2-11e5-8da2-3d7e76222f7a.png)

the textfield size is incorrect

**Comment by tobiasgr** [comment]
+1, in a swift project it is enough to just add the pod to the project, you do not even have to do number 5.

**Comment by tobiasgr** [comment]
A work-around:

collection-view collection-view-cell > view{
    width: 270px;
}

**Comment by markddaa** [comment]
works on alert textfield but also affects on collection view cell.....

**Comment by tobiasgr** [comment]
Yeah, skipped this workaround myself, since it conflicted with uiactivityviewcontroller, and rewrote the app to not contain any alerts with input fields. I guess Pixate Freestyle is kind of dead...

**Comment by Pitsko** [comment]
one more workaround

var list = alertController.view.subviews as! [UIView]
               for _ in 0 ..< 7 {
                   list = list.flatMap { (view) in
                       return view.subviews as! [UIView]
                   }
               }

               for view in list {
                   if view is UICollectionView {
                       view.styleClass = "fix_text_field_in_alert_view"
                   }
               }

where
.fix_text_field_in_alert_view collection-view-cell > view {
    width: 270px;
}

**Comment by markddaa** [comment]
@Pitsko that works! thx so much for the provided workaround.
Hopefully Pixate will update their product to support iOS 9 soon
